app/controllers/admin/community_customizations_controller.rb

```diff
@@ -23,16 +28,24 @@ def update_details
         :transaction_agreement_content
       ]
       params.require(:community_customizations).require(locale).permit(*permitted_params)
-
       locale_params = params[:community_customizations][locale]
       customizations = find_or_initialize_customizations_for_locale(locale)
       customizations.update_attributes(locale_params)
     end
 
+    enabled_locales = params[:enabled_locales]
+    all_locales = MarketplaceService::API::Marketplaces.all_locales
+    enabled_locales_valid = enabled_locales.map do |locale|
+      all_locales.include? locale
+    end
+    if enabled_locales_valid
```

Comment:
Because `map` always returns an array, the `enabled_locales_valid` will be an array of boolean values (or empty array). In Ruby, everything else than `false` and `nil` is false, so this `if` block will always be executed.

I think what you are looking for is `all?` method. It returns `true` if all the items in array return true, so:

```ruby
enabled_locales_valid = enabled_locales.all? { all_locales.include?(locale) }
```